src/Uno.Extensions.Templates/content/unoapp-extensions/azure-pipelines.yml

```diff
@@ -0,0 +1,256 @@
+trigger:
+- master
+- main
+- release
+
+# trigger this pipeline if there's a PR to any of these branches
+pr:
+- master
+- main
+- staging
+- releases/*
+
+parameters:
+- name: Wasm
+  type: string
+  default: 'true'
+- name: Windows
+  type: string
+  default: 'true'
+- name: WPF
+  type: string
+  default: 'true'
+- name: Linux
+  type: string
+  default: 'true'
+- name: GTK
+  type: string
+  default: 'true'
+- name: Android
+  type: string
+  default: 'true'
+- name: iOS
+  type: string
+  default: 'true'
+- name: MacCatalyst
+  type: string
+  default: 'true'
+
+variables:
+- name: jobTimeout
+  value: '40'
+
+- name: dotnet_version
+  value: '7.0.102'
+
+stages:
+  - stage: Build_UNO
+
+    jobs:
+    - job: Wasm
+      condition:  eq('${{ parameters.Wasm }}', 'true')
+      timeoutInMinutes: '${{ variables.jobTimeout }}'
+      steps:
+        - template: /azure/steps/install-dependencies.yml  
+          parameters:
+            dotnet_version: '${{ variables.dotnet_version }}'
+            target_platform: 'wasm'
+
+        - template: /azure/steps/wasm.yml
+          parameters:
+            package_id: 'Wasm'
+            project_folder: 'MyExtensionsApp.Wasm'
+            project_to_build: 'MyExtensionsApp.Wasm/MyExtensionsApp.Wasm.csproj'
+      pool:
+        vmImage: macOS-latest   
+
+    - job: Android
+      condition:  eq('${{ parameters.Android }}', 'true')
+      timeoutInMinutes: '${{ variables.jobTimeout }}'
+      steps:
+        - template: /azure/steps/install-dependencies.yml  
+          parameters:
+            dotnet_version: '${{ variables.dotnet_version }}'
```

Comment:
The parameter value matches the default, so we can remove this from the azure-pipeline.yml

Reply:
This comment applies to all parameters where the value being passed in matches the default - let's keep this file as simple as possible